libs/model/src/contest/Contests.projection.ts

```diff
@@ -52,22 +91,41 @@ export const Contests: Projection<typeof inputs> = () => ({
     },
 
     ContestContentAdded: async ({ payload }) => {
+      // TODO: can we make this just one sql statement using subqueries?
```

Comment:
Any reason not to do this as part of this PR? This would either need to be a completely raw query or a [Sequelize literal](https://sequelize.org/docs/v6/other-topics/sub-queries/) (not sure if literals are supported in `.create` func cal).

Reply:
I'm leaving TODOs in sections that could use some convention. Unsure if there's an existing one I'm not aware of, or if we need to brainstorm the optimal pattern. If there's a convention in place, I'll address it in this PR.

Reply:
I don't think we have a convention but I will almost always be in favor of performance gains. I think if you don't want to address this as part of this PR we should open a follow-up ticket to modify this in the near future since it is quite an easy optimization.

Reply:
Agree. I would like to discuss the convention first, and then plan for searching and refactoring existing code. Will add ticket to refactoring bucket.